browser/brave_rewards/android/brave_rewards_native_worker.cc

```diff
@@ -701,19 +701,23 @@ void BraveRewardsNativeWorker::OnGetExternalWallet(
     const ledger::type::Result result,
     ledger::type::UpholdWalletPtr wallet) {
   std::string json_wallet;
-  base::Value dict(base::Value::Type::DICTIONARY);
-  dict.SetStringKey("token", wallet->token);
-  dict.SetStringKey("address", wallet->address);
-
-  // enum class WalletStatus : int32_t
-  dict.SetIntKey("status", static_cast<int32_t>(wallet->status));
-  dict.SetStringKey("verify_url", wallet->verify_url);
-  dict.SetStringKey("add_url", wallet->add_url);
-  dict.SetStringKey("withdraw_url", wallet->withdraw_url);
-  dict.SetStringKey("user_name", wallet->user_name);
-  dict.SetStringKey("account_url", wallet->account_url);
-  dict.SetStringKey("login_url", wallet->login_url);
-  base::JSONWriter::Write(dict, &json_wallet);
+  if (!wallet) {
```

Comment:
@SergeyZhukovsky i think this change has already landed on 1.17 and 1.18. this PR is only for 1.16 right ?

Reply:
@deeppandya correct, `1.17.x` & `1.18.x` works as expected. Only being merged into `1.16.x`.